### ISSUE TITLE
Unpublished Models in User's Profile

### ISSUE THREAD

**Issue by seanbergin** [comses/comses.net#587]
I noticed that unpublished models are visible in the list of models published by a user. I was looking at https://www.comses.net/users/972/ - and saw several unpublished models listed in the models - tab. I checked and these unpublished models are viewable when not logged in ( so I don't have some sort of special permissions to see them). Obviously, the unpublished models should not be published here or they become visible to everyone very easily.

**Comment by alee** [comment]
Good find, thanks Sean!

I agree they shouldn't show up in that list but can you verify that you are unable to access the actual details of the model if you click on the link? It should go to the comses "resource not found" page.  As it stands it just displays a list item style metadata about the model.

**Comment by seanbergin** [comment]
You are correct, I hadn't noticed that it directs you to a resource not found page. So its much less of a problem. The only other potential issue is that if a paper is under peer review somewhere, or under review on comses, it does make it simple to find the author of a model. I'm not sure that is something that is likely though - there are probably another half dozen ways people can identify the author of a paper thanks to the world wide web.
